airflow/www/decorators.py

```diff
@@ -44,7 +45,7 @@ def wrapper(*args, **kwargs):
                 event=f.__name__,
                 task_instance=None,
                 owner=user,
-                extra=str(list(request.values.items())),
+                extra=json.dumps(request.values),
```

Comment:
Why not fetch str and handle serialization on the client-side?

Reply:
The current setup doesn't produce valid json, because it is a list of tuples.  It would also be very helpful to be able to do queries in the database, since important context lives in the `extra`, eg whether or not a clear was confirmed, or a trigger had a `conf`.

Reply:
@kaxil to give more color, we've built an "Airflow Snitch" which alerts based on some user actions (eg user cleared a production DAG).  This relies on parsing the `extra` field to determine if the job was actually cleared since there are two trigger events, and only the `confirmed` event causes an action.

We've hacked together some `extra` field parsing, but it isn't very elegant, and this could be done a lot more efficiently in the database if the column was valid json.

Reply:
Thanks for the explanation, can you add a test, please